speakers.js

```diff
@@ -0,0 +1,169 @@
+const container = document.createElement('section');
```

Comment:
In this file, you should save your speakers in an array, just the way it was done in the previously submitted [repo](https://github.com/hafedEfheij/Capstone-Project-1/blob/main/assets/javascript/speakers.js) so that you can easily add more speakers in the future.